docs/examples/rbac/go/Makefile

```
@@ -1,4 +1,5 @@
 .PHONY: test
 
 test:
+	make -C ../../../../languages/go
```

Maybe worth specifying the (current) default task in case we change it? Looks like it's `copy_lib`:
```suggestion
	make -C ../../../../languages/go copy_lib
```
Also begs the question... what happens if we run `make -C docs/examples/rbac/go test` before we build the core lib? Should we add a task in the Go makefile that builds the Rust core?

---

Oh yeah actually wasn't sure about this. Will change to `make copy_lib`. 

And yeah we probably should be explicit somehow that this depends on the rust core being built. In this case it happens to be built by the earlier python build.

Through this change I have been feeling like we should do a little audit of all of our make dependencies. I bet we could clean a lot up and also improve build times if we really nail down what depends on what, especially in the docs tests.